Mail/Views/Attachment/AttachmentPreview.swift

```diff
@@ -72,7 +74,7 @@ struct AttachmentPreview: View {
         @InjectService var matomo: MatomoUtils
         matomo.track(eventWithCategory: .message, name: "download")
         guard let url = attachment.localUrl,
-              var source = UIApplication.shared.mainSceneKeyWindow?.rootViewController else {
+              var source = rootViewControllerFetcher.rootViewController else {
```

Comment:
We keep it like this for now but ideally I would prefer getting the current view controller from elsewhere. Eg. UIViewRepresentable

Reply:
Sure, I only wrapped tge `UIApplication` call in a protocol as it is prohibited when building in ext.